cpu/k64f/Makefile.include

```diff
@@ -0,0 +1,28 @@
+# define the CPU architecture for the k64f
+export CPU_ARCH = cortex-m4
+
+# tell the build system that the CPU depends on the Kinetis common files
+export USEMODULE += kinetis_common
+
+# define path to kinetis module, which is needed for this CPU
+export KINETIS_COMMON = $(RIOTCPU)/kinetis_common/
+# CPU depends on the kinetis module, so include it
+include $(KINETIS_COMMON)Makefile.include
+
+export LINKFLAGS += -L$(RIOTCPU)/kinetis_common/ldscripts
+# define the linker script to use for this CPU
+export LINKERSCRIPT = $(CPU_MODEL).ld
```

Comment:
I think you can drop this line, as the linkerscript is defined in `boards/Makefile.include.cortexm_comon`